Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -27,6 +27,10 @@ namespace AzToolsFramework
 {
     namespace AssetBrowser
     {
+        const float MIN_HEADER_RESIZE_PROPORTION = .25f;
+        const float MAX_HEADER_RESIZE_PROPORTION = .75f;
+        const float DEFAULT_HEADER_RESIZE_PROPORTION = .5f;
```

Comment:
update: The convention for constants is `UpperCamelCase` (reserve uppercase only for macros)